src/encord_active/lib/project/project.py

```diff
@@ -92,6 +93,38 @@ def from_encord_project(self, encord_project: EncordProject) -> Project:
 
         return self.load()
 
+    def refresh(self):
+        """
+        Refresh project data and labels using its remote project in Encord Annotate.
+
+        :return: The updated project instance.
+        """
+
+        if not self.project_meta.get("has_remote", False):
+            raise AttributeError("The project does not have a remote project associated to it.")
+
+        project_hash = self.project_meta.get("project_hash")
+        if project_hash is None:
+            raise AttributeError(
+                "The project metadata is missing the reference to the remote project in Encord Annotate. "
+                "Add the project hash of the remote project to the property `project_hash` in the file "
+                f"{self.file_structure.project_meta}."
+            )
+
+        ssh_key_path = self.project_meta.get("ssh_key_path")
+        if ssh_key_path is None:
+            raise AttributeError(
+                "The project metadata is missing the path to the private SSH key needed to log into Encord Annotate. "
+                f"Add such path to the property `ssh_key_path` in the file {self.file_structure.project_meta}."
+            )
```

Comment:
There is a utility function somewhere to both look for the ssh key in the project meta and in the EA settings.

Reply:
while I didn't find the right utility function I added the same behaviour in the `dict.get()` :+1: